src/cdc/runner.zig

```diff
@@ -221,14 +221,17 @@ pub const Runner = struct {
         self.message_pool = try MessagePool.init(allocator, .client);
         errdefer self.message_pool.deinit(allocator);
 
-        self.vsr_client = try Client.init(allocator, .{
-            .id = stdx.unique_u128(),
-            .cluster = options.cluster_id,
-            .replica_count = @intCast(options.addresses.len),
-            .time = time,
-            .message_pool = &self.message_pool,
-            .message_bus_options = .{ .configuration = options.addresses, .io = &self.io },
-        });
+        self.vsr_client = try Client.init(
+            allocator,
+            &self.message_pool,
+            time,
```

Comment:
Nit, but I would probably swap the order of these arguments, so that we go from "most general -> most specific".